### ISSUE TITLE
Remove hard-coded LandTools.php references to allow for configuration by grid operator

### ISSUE THREAD

**Issue by emperorstarfinder** [Virtual-Universe/Universe-Viewer-Dev#12]
Viewers currently have LandTools.php hard-coded within the source code.  This causes grids to need to use a redirection process in their web servers (i.e. apache or nginx) to redirect to the page format they are using such as .html, .aspx, etc.

We should remove this to allow the viewer to be more friendly with each grid.  This will allow for the viewer to communicate on the protocol used by each individual grid should they choose to change things on the grid architecture.

Emperor Starfinder
Founder
Chief Executive Officer
Core Developer
Governing Body Chair
Grid Architecture Developer
Work Group Leader
Grid Architecture Development Work Group
Second Galaxy Development Team
Email: emperor@secondgalaxy.com

**Comment by CaseyraeStarfinder** [comment]
I agree with this enhancement.  We will add this enhancement into the beta release.

Best Regards,

Caseyrae Starfinder
Governing Body Vice Chair (Europe)
Core Developer
Viewer Developer Work Group Leader
Viewer Development Work Group
Second Galaxy Development Team
Email: caseyrae@secondgalaxy.com
